linkup-cli/src/commands/deploy/cf_deploy.rs

```diff
@@ -58,9 +58,12 @@ pub async fn deploy(args: &DeployArgs) -> Result<(), DeployError> {
     );
     let notifier = ConsoleNotifier::new();
 
+    let tunnel_zone_name = cloudflare_api.get_zone_name(&args.zone_ids[0]).await?;
```

Comment:
Why tunnel zone name and not like zone1name-zone2name list? the script is "global" and will be associated with "all" the zones / have worker routes in several places

Reply:
No specific reason! Just thought maybe it was enough.
Is this better? e71238d88ee38123143cb51ca0640dff7e453cf8